config/tgl-cavs.toml

```diff
@@ -266,3 +266,22 @@ count = 9
 		20, 0, 0, 0, 12832, 20881000, 0, 0, 0, 20881, 0,
 		21, 0, 0, 0, 12832, 23431000, 0, 0, 0, 23431, 0,
 		22, 0, 0, 0, 12832, 30471000, 0, 0, 0, 30471, 0]
+
+	[[module.entry]]
+	name = "SMTAMP"
+	uuid = "167A961E-8AE4-11EA-89F1-000C29CE1635"
```

Comment:
it should be clearer that this is a test module....

Reply:
@aiChaoSONG change the name and add some toml comments to indicate this is a testing module

Reply:
updated! the max len for the module name is 8(include ending '\0'), so I will call it SMATEST